Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -1212,6 +1220,97 @@ public override void PruneOutdatedChains(bool noopWithoutCanon = false)
             batch.Clear();
         }
 
+        public override BlockCommit? GetLastCommit(long height)
+        {
+            _rwLastCommitLock.EnterReadLock();
+
+            try
+            {
+                byte[] key = LastCommitKey(height);
+                byte[] bytes = _lastCommitDb.Get(key);
+                if (bytes is null)
+                {
+                    return null;
+                }
+
+                return new BlockCommit(bytes);
+            }
+            catch (Exception e)
+            {
+                LogUnexpectedException(nameof(GetTxNonce), e);
+            }
+            finally
+            {
+                _rwLastCommitLock.ExitReadLock();
+            }
+
+            return null;
+        }
+
+        public override void PutLastCommit(BlockCommit lastCommit)
+        {
+            byte[] key = LastCommitKey(lastCommit.Height);
+
+            if (!(_lastCommitDb.Get(key) is null))
+            {
+                DeleteLastCommit(lastCommit.Height);
+            }
```

Comment:
```suggestion
            if (_lastCommitDb.Get(key) is { })
            {
                DeleteLastCommit(lastCommit.Height);
            }
```
Also, is silently overwriting a stored last commit a good idea? 🤔

Reply:
Oops, I misunderstood that height could be overwritten by round.